docker-compose/docker-compose-hcd.yml

```diff
@@ -22,6 +22,50 @@ services:
       timeout: 10s
       retries: 20
 
+  hcd-2:
```

Comment:
Do we want to do change set up for HCD to run 3 nodes (instead of 1)?
I understand it's good to have parity, but startup time for single node setup is very nice.

Reply:
Ahhh. There's `-n` switch for number of nodes, just found out.